### PR TITLE
Provide messageArguments for exceptions

### PR DESCRIPTION
**Pull request by palagdan**
Resolves partially #kbss-cvut/fta-fmea-ui#550

### CODE REVIEW

**Review by blcham — CHANGES_REQUESTED** [review]
LGTM, just see my comments

File: src/main/java/cz/cvut/kbss/analysis/model/fta/FTAMinimalCutSetEvaluation.java [hunk 1]
```diff
@@ -35,9 +35,9 @@ public void evaluate(FaultTree faultTree){
         if(!fe.isEmpty()){
             String message =
                     fe.stream().map(e -> "'%s'".formatted(e.getName()))
-                            .collect(Collectors.joining("\n"  , "The following leaf events do not have specified probability: [\n", "]"));
+                            .collect(Collectors.joining("\n"));
```
[the remark this answers is not in the document]
Comment (palagdan) [reply src/main/java/cz/cvut/kbss/analysis/model/fta/FTAMinimalCutSetEvaluation.java:38]: No, it wasn't just for the default message. It looked correct on the frontend. I've fixed it.

Screenshot from 2024-09-18 11-31-44

File: src/main/java/cz/cvut/kbss/analysis/service/SystemRepositoryService.java [hunk 2]
```diff
@@ -68,7 +69,13 @@ public System findRequired(URI id) {
     }
 
     @Transactional
-    public System create(System system){
+    public System create(System system) {
+        List<URI> existingSystems = systemDao.findUriByName(system.getName());
+        if (!existingSystems.isEmpty())
+            throw new LogicViolationException("error.system.nameExists",
+                    ("System with name\"%s\" already exists")
```
[the remark this answers is not in the document]
Comment (palagdan) [reply src/main/java/cz/cvut/kbss/analysis/service/SystemRepositoryService.java]: done

File: src/main/java/cz/cvut/kbss/analysis/model/fta/FTAMinimalCutSetEvaluation.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (blcham) [reply src/main/java/cz/cvut/kbss/analysis/model/fta/FTAMinimalCutSetEvaluation.java:38]: ok

**Review by blcham — APPROVED** [review]

**Pull request merged**